### PR TITLE
add rf notebook examples and some missing rf setters

### PR DESCRIPTION
**Pull request by eordentlich**

### CODE REVIEW

File: notebooks/random-forest-classification.ipynb [hunk 1]
```diff
@@ -0,0 +1,640 @@
...
+   "outputs": [],
+   "source": [
+    "df_train.printSchema()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### We will use this function to build both the Spark RAPIDS ML (GPU) and Spark ML (CPU) random forest classifier objects, demonstrating the common API, and verify they yield similar performance on our synthetic dataset"
```
Comment (leewyang) [inline notebooks/random-forest-classification.ipynb:110]: This is a very long section header.  Maybe shorten and move most of text into body instead?

File: notebooks/random-forest-regression.ipynb [hunk 2]
```diff
@@ -0,0 +1,658 @@
...
+   "outputs": [],
+   "source": [
+    "df_train.printSchema()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### We will use this function to build both the Spark RAPIDS ML (GPU) and Spark ML (CPU) random forest regressor objects, demonstrating the common API, and verify they yield similar performance on our synthetic dataset"
```
Comment (leewyang) [inline notebooks/random-forest-regression.ipynb:110]: Same here.

**Review by leewyang — COMMENTED** [review]
Minor comments, otherwise LGTM.  
@wbo4958 should probably also review.

File: notebooks/random-forest-classification.ipynb [hunk 3]
```diff
@@ -0,0 +1,640 @@
...
+     "cellMetadata": {},
+     "inputWidgets": {},
+     "nuid": "a7c21330-7326-4d98-9351-d1b2e4c6143c",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
```
Comment (lijinf2) [inline notebooks/random-forest-classification.ipynb:24]: Seems numpy and np are not used.

**Comment by lijinf2** [comment]
Looks nice!

**Comment by eordentlich** [comment]
build

File: notebooks/random-forest-regression.ipynb (hunk 2 above)
Comment (eordentlich) [reply notebooks/random-forest-regression.ipynb]: Done

File: notebooks/random-forest-classification.ipynb (hunk 3 above)
Comment (eordentlich) [reply notebooks/random-forest-classification.ipynb]: Removed

**Review by leewyang — APPROVED** [review]

**Pull request merged**